plugin/src/sidebar.ts

```diff
@@ -584,12 +584,14 @@ function createLayer(dataset: DataCatalogItem, overrides?: any) {
             style: "point",
             pointSize: 10,
             pointColor: "white",
+            clampToGround: true,
```

Comment:
About marker, we need to use heightReference.

```suggestion
            heightReference: "clamp",
```

Reply:
Done

Reply:
83e66e860c7ad90109b6c1185f6cdcbb085c5d10